client/src/components/Modal.tsx

```diff
@@ -0,0 +1,44 @@
+import {
+  AlertDialog,
+  AlertDialogAction,
+  AlertDialogCancel,
+  AlertDialogContent,
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  AlertDialogTrigger,
+} from './ui/alert-dialog';
+import { ReactNode } from 'react';
+
+type ModalProps = {
+  children: ReactNode;
+  onContClick: () => Promise<void>;
+  deleteName: string;
+};
+
+export function Modal({ children, deleteName, onContClick }: ModalProps) {
+  return (
+    <AlertDialog>
+      <AlertDialogTrigger asChild>{children}</AlertDialogTrigger>
+      <AlertDialogContent>
+        <AlertDialogHeader>
+          <AlertDialogTitle>Are you absolutely sure?</AlertDialogTitle>
+          <AlertDialogDescription>
+            This action cannot be undone. This will permanently delete your{' '}
```

Comment:
Since there are spaces on the next line, the `{' '}` should be unnecessary. (Though sometimes JSX surprises me.)

Same with the one on line 31. The formatter will put those in to make sure it does not modify the output behavior, but frequently the ones it puts in are unnecessary.